LoopKitUI/InsulinKit/InsulinDeliveryTableViewController.swift

```diff
@@ -365,14 +381,15 @@ public final class InsulinDeliveryTableViewController: UITableViewController {
                 let time = timeFormatter.string(from: entry.startDate)
 
                 cell.textLabel?.text = "\(volume) U"
```

Comment:
[nit] I know this wasn't changed in this PR, but should this be a localized string?